src/js/libgeo/GeoOps.js

```diff
@@ -1724,12 +1737,68 @@ geoOps.TrMoebiusP.kind = "P";
 geoOps.TrMoebiusP.updatePosition = function(el) {
     var t = csgeo.csnames[(el.args[0])];
     var p = csgeo.csnames[(el.args[1])].homog;
-    var l1 = List.productMV(t.mat1, p);
-    var l2 = List.productMV(t.mat2, p);
-    el.homog = List.normalizeMax(List.cross(l1, l2));
+
+    el.homog = geoOps._helper.TrMoebiusP(p, t.mat1, t.mat2);
     el.homog = General.withUsage(el.homog, "Point");
 };
 
+geoOps._helper.TrMoebiusP = function(p, mat1, mat2) {
+    var l1 = List.productMV(mat1, p);
+    var l2 = List.productMV(mat2, p);
+    return List.normalizeMax(List.cross(l1, l2));
+};
```

Comment:
I'd suggest you pass in the Möbius transformation as a complete object, instead of passing the two matrices in separately. Everyone calling this code will be holding both in any case, so splitting things up only here means the rest of the code won't have to care about how a Möbius transformation is actually performed internally.